buildAndReleaseTask/task.json

```diff
@@ -68,6 +68,14 @@
             "defaultValue": "",
             "helpMarkDown": "The Pulumi Stack against which to run the commands.",
             "required": "true"
+        },
+        {
+            "name": "expectedVersion",
```

Comment:
Instead of `expectedVersion`, can we call this `versionSpec`, perhaps? It'll be similar to the built-in `NodeTool` task.

Reply:
done